### DISCUSSION TITLE
msfs 2020 stop working 30/4 when start flybywire neo320..

### DISCUSSION THREAD

**Discussion by akwa57** [flybywiresim/aircraft#4713] [Help] [answered]
msfs 2020 stop working 30/4 when start flybywire neo320..

**Comment by MapleNAV-TCASFAULT** [comment, marked as answer]
can you provide more detail, instead of just saying that it is not working.

**Reply by akwa57** [reply]
is anybody more report issues when windows 10 update something in 30april?
it crash under loading procedur time after choice airplane

**Reply by MapleNAV-TCASFAULT** [reply]
I had a recent windows update too, which kinda messed up my PC but still didn't make my sim unplayable. I suggest you to remove everything from your community folder and reinstall the a32nx.

**Comment by akwa57** [comment]
Now it seems work after new update windows yesterday an reinstall flight sim and flybywire


Skickades från E-post<https://go.microsoft.com/fwlink/?LinkId=550986> för Windows 10

Från: ***@***.***>
Skickat: den 5 maj 2021 01:06
Till: ***@***.***>
Kopia: ***@***.***>; ***@***.***>
Ämne: Re: [flybywiresim/a32nx] msfs 2020 stop working 30/4 when start flybywire neo320.. (#4713)


I had a recent windows update too, which kinda messed up my PC but still didn't make my sim unplayable. I suggest you to remove everything from your community folder and reinstall the a32nx.
